client/my-sites/comments/main.jsx

```diff
@@ -10,21 +10,33 @@ import { localize } from 'i18n-calypso';
 import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
+import CommentList from './comment-list';
+import CommentNavigation from './comment-navigation';
 
 export class CommentsManagement extends Component {
 
 	static propTypes = {
 		basePath: PropTypes.string,
+		siteSlug: PropTypes.string,
+		status: PropTypes.string,
 		translate: PropTypes.func,
 	};
 
 	render() {
-		const { translate, basePath } = this.props;
+		const {
+			basePath,
+			siteSlug,
+			status,
+			translate,
+		} = this.props;
 		return (
-			<Main className="comments">
+			<Main className="comments" wideLayout>
 				<PageViewTracker path={ basePath } title="Manage Comments" />
 				<DocumentHead title={ translate( 'Manage Comments' ) } />
-				<div>Hello World!</div>
+				<div className="comments__primary">
+					<CommentNavigation siteSlug={ siteSlug } status={ status } />
+					<CommentList />
```

Comment:
well, shouldn't we justify why we need an infinite list before we toss it in there?

seems like we could quickly get carried away with it here considering how many comments there might be.

one thing I worry about with an infinite list here is that we introduce jack when we might have no reason to. the _number_ and _id_ of each comment is a simple and fast database query which could pull in the full list of comments (and even build the tree if we wanted). _however_, getting any amount of comment content in really starts bogging us down performance-wise. if we can load the entire "empty" list of comments up front with a quick query and fill in as we go I think we could have a much better experience